plugin/src/main/scala/com/github/sbt/jni/build/CMake.scala

```diff
@@ -37,14 +37,14 @@ object CMake extends BuildTool with ConfigureMakeInstall {
       if (cmakeVersion >= 312) Seq("--parallel", parallelJobs.toString())
       else Seq.empty
 
-    override def configure(target: File) = {
+    override def configure(target: File) = cmakeProcess(
       // disable producing versioned library files, not needed for fat jars
-      val args = Seq(s"-DCMAKE_INSTALL_PREFIX:PATH=${target.getAbsolutePath}", "-DCMAKE_BUILD_TYPE=Release", "-DSBT:BOOLEAN=true") ++ scala.sys.props
-        .get("java.home")
-        .toSeq
-        .map(jh => s"-DJAVA_HOME=${jh}") ++ Seq(cmakeVersion.toString, baseDirectory.getAbsolutePath)
-      cmakeProcess(args: _*)
-    }
+      s"-DCMAKE_INSTALL_PREFIX:PATH=${target.getAbsolutePath}",
+      "-DCMAKE_BUILD_TYPE=Release",
+      "-DSBT:BOOLEAN=true",
+      cmakeVersion.toString,
+      baseDirectory.getAbsolutePath
+    )
```

Comment:
@berkeleybarry sadly I had to rollback this change. It looks like this or should be made somehow configurable for the client in a separate PR or smth else 🤔 
CI definitely doesn't work with the change. 
I searched for a bit, and it looks like it is not a regular practice. 

May be setting an env variable may work for you?